sdk/program/src/program_error.rs

```diff
@@ -117,6 +120,7 @@ pub const BORSH_IO_ERROR: u64 = to_builtin!(15);
 pub const ACCOUNT_NOT_RENT_EXEMPT: u64 = to_builtin!(16);
 pub const UNSUPPORTED_SYSVAR: u64 = to_builtin!(17);
 pub const ILLEGAL_OWNER: u64 = to_builtin!(18);
+pub const ACCOUNTS_DATA_BUDGET_EXCEEDED: u64 = to_builtin!(19);
 // Warning: Any new program errors added here must also be:
```

Comment:
Down on line 127 (GitHub wouldn't let me comment directly there...), it says
> - Be featureized in the BPF loader to return `InstructionError::InvalidError`

Is there anything I need to do for that specifically? It sounds like yes, but I wasn't sure where/how exactly.

Reply:
@jackcmay or @Lichtso

Reply:
Check out the commit that added that comment: https://github.com/solana-labs/solana/commit/83b9a046d194e8ba30f1deb1f96b153366e7a271

Reply:
Done! Please see https://github.com/solana-labs/solana/pull/21813#discussion_r774109657